blueprints/cloud-operations/network-dashboard/dashboards/quotas-utilization.json

```diff
@@ -375,31 +411,34 @@
           }
         },
         "width": 6,
-        "yPos": 20
+        "xPos": 6,
+        "yPos": 16
       },
       {
         "height": 4,
         "widget": {
-          "title": "ip_addresses_per_subnet_utilization",
```

Comment:
Could you please add the ip_addresses_per_subnet_utilization metric in the dashboard that you removed?